lib/localization/locales/tr.json

```
@@ -1,20 +1,20 @@
 {
-  "loading": "Yükleniyor...",
+  "loading": "Yükleniyor…",
```

Please don't use a unicode ellipses.

```suggestion
  "loading": "Yükleniyor...",
```